app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -848,17 +847,26 @@ export default Ember.Controller.extend({
             return self.confirmDraftAbandon(data);
           })
           .then(data => {
-            opts.draft = opts.draft || data.draft;
-
-            // we need a draft sequence for the composer to work
-            if (opts.draft_sequence === undefined) {
-              opts.draftSequence = data.draft_sequence;
+            if (!opts.draft && data.draft) {
+              opts.draft = data.draft;
             }
-
+            opts.draftSequence = data.draft_sequence;
             self._setModel(composerModel, opts);
           })
           .then(resolve, reject);
       }
+      // otherwise, do the draft check async
+      else if (!opts.draft && !opts.skipDraftCheck) {
+        Draft.get(opts.draftKey)
```

Comment:
technically there is a timing issue here, if get for draft is mega delayed

Reply:
That was pretty much the point of these changes: open composer asap and do not wait for draft conflict check, because the response might be delayed.